.github/workflows/linux_riscv.yml

```diff
@@ -160,8 +166,8 @@ jobs:
             -DCMAKE_VERBOSE_MAKEFILE=ON \
             -DCMAKE_COMPILE_WARNING_AS_ERROR=OFF \
             -DCMAKE_TOOLCHAIN_FILE=${OPENVINO_BUILD_DIR}/dependencies/conan_toolchain.cmake \
-            -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
-            -DCMAKE_C_COMPILER_LAUNCHER=ccache \
+            -DCMAKE_CXX_COMPILER_LAUNCHER=${{ env.CMAKE_CXX_COMPILER_LAUNCHER }} \
+            -DCMAKE_C_COMPILER_LAUNCHER=${{ env.CMAKE_C_COMPILER_LAUNCHER }} \
```

Comment:
sccache does not work for this job https://github.com/openvinotoolkit/openvino/actions/runs/6551829580/job/17793841318?pr=20484

Reply:
I need to investigate this further, I clearly see that sccache is called during the compilation, but it still shows 0 compilation requests at the end.

Reply:
ok, let's just create a ticket to not forget about it and move further

Reply:
So, something interesting is going on there. I've configured `sccache` to use empty local directory, and here's the result:
- Build time - 1h 26m 13s
- `sscache` stats:
```
Run ${SCCACHE_PATH} --show-stats
Compile requests                      0
Compile requests executed             0
Cache hits                            0
Cache misses                          0
Cache timeouts                        0
Cache read errors                     0
Forced recaches                       0
Cache write errors                    0
Compilation failures                  0
Cache errors                          0
Non-cacheable compilations            0
Non-cacheable calls                   0
Non-compilation calls                 0
Unsupported compiler calls            0
Average cache write               0.000 s
Average compiler                  0.000 s
Average cache read hit            0.000 s
Failed distributed compilations       0
Cache location                  Local disk: "/github/home/.cache/sccache"
Version (client)                0.5.4
```

And with Azure Blob Storage enabled:
- Build time - 35m 15s
- `sccache` stats:
``` 
Run ${SCCACHE_PATH} --show-stats
Compile requests                      0
Compile requests executed             0
Cache hits                            0
Cache misses                          0
Cache timeouts                        0
Cache read errors                     0
Forced recaches                       0
Cache write errors                    0
Compilation failures                  0
Cache errors                          0
Non-cacheable compilations            0
Non-cacheable calls                   0
Non-compilation calls                 0
Unsupported compiler calls            0
Average cache write               0.000 s
Average compiler                  0.000 s
Average cache read hit            0.000 s
Failed distributed compilations       0
Cache location                  azblob, name: sccache, prefix: /ubuntu22_riscv64_Release/
Version (client)                0.5.4
```

So there IS a cache, we just don't know why the build was so slow (may be because it wasn't a 100% cache hit rate?)

Reply:
Suggest to keep ccache for RISC-V build temporary.
I know the reason and it will take some time to investigate

Reply:
I've reverted changes for this workflow, we can leave it with `ccache` and deal with it later